ako-operator/helm/ako-operator/crds/ako.vmware.com_akoconfigs.yaml

```diff
@@ -117,10 +138,6 @@ spec:
                     description: TenantName is the name of the tenant where all AKO
                       objects will be created in Avi.
                     type: string
-                  tenantsPerCluster:
```

Comment:
@DixitAakash : Do not remove existing properties.

Reply:
Added back.